src/Molecules/Footer/FooterStyled.js

```diff
@@ -0,0 +1,31 @@
+import styled from 'styled-components';
+
+export const Wrapper = styled.div`
+  display: block;
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  display: flex;
+  flex-direction: flex;
+  justify-content: space-between;
+  margin: 0 25px 0 25px;
+  height: 77px;
+  border-top: 2px solid #f5f7fa;
+`;
+
+export const LogoWrapper = styled.div`
+  display: flex;
+  pointer-events: none;
```

Comment:
If there shouldn't be possibility to click at this wrapper keep in mind that pointer-events property is not working with keyboard navigation.